CHANGELOG.md

```diff
@@ -16,6 +16,24 @@ and this project adheres to
 
 ### Added
 
+- Added `.toMatchStepMetadata` jest matcher. This matcher complements the
+  `executeStepWithDependencies` utility. Usage:
+
+  ```ts
+  const stepResult = await executeStepWithDependencies({
+    stepId: Steps.FETCH_USERS.id,
+    invocationConfig,
+    instanceConfig,
+  });
+
+  expect(stepResult).toMatchStepMetadata({
+    stepId: Steps.FETCH_USERS.id,
+    invocationConfig,
+  });
+  ```
+
+- Added `StepTestConfig` and `StepTestConfigWithCredentials` exports from
+  `@jupiterone/integration-sdk-testing`
```

Comment:
I'm unsure of the significance of these now being exported. Are they meant to be used by integrations, so that this should indicate what the user is to do with them now that they're exported?

Reply:
It's not a _super_ significant change, but it is an addition so I figured it'd be worth adding to the changelog.

Here's how I see using this:

```ts
const config: StepTestConfigWithCredentails = {
  stepId: 'my-step-id',
  invocationConfig,
  instanceConfig,
};

const stepResult = await executeStepWithDependencies(config);

expect(stepResult).toMatchStepMetadata(config);
```